### ISSUE TITLE
[TextFields] Placeholder text color doesn't seem to pass contrast requirements on white background

### ISSUE THREAD

**Issue by jmdetloff** [material-components/material-components-ios#4324]
Testing with: https://webaim.org/resources/contrastchecker/ shows a failing contrast ratio for placeholders.

**Comment by jverkoey** [comment]
We need to get design to provide us colors that pass contrast requirements.

**Comment by andrewoverton** [comment]
I got rid of all custom theming in Catalog and checked the contrast between the placeholder text and background and found that with the default color scheme we actually do meet contrast requirements. @romoore says this is sufficient, so I will close the issue.
